### DISCUSSION TITLE
Does markers has the right color?

### DISCUSSION THREAD

**Discussion by acarmona1989** [apexcharts/apexcharts.js#2622] [Q&A]
I would like to know why Team B  marker has the color used by serie Team A? Is it a usual practice in chart representation or it could be considered a bug? 
![image](https://user-images.githubusercontent.com/8333232/133329914-feed8ca4-4dba-4a95-8815-2744e7e2d490.png)

The screenshot was taken from this link https://apexcharts.com/javascript-chart-demos/mixed-charts/line-column-area/

**Comment by junedchhipa** [comment]
That looks wrong. Can you please open an issue with `bug` label?
